pkg/tar/tar.go

```diff
@@ -206,6 +206,15 @@ func extractFile(tr *tar.Reader, target string, hdr *tar.Header, overwrite bool,
 		}
 	}
 
+	if typ == tar.TypeReg || typ == tar.TypeRegA {
+		for k, v := range hdr.Xattrs {
+			err := syscall.Setxattr(p, k, []byte(v), 0)
```

Comment:
Not specific cases from off top of my head, but I can imagine several misuses. I'd say it would be safer here to perform some prefix-whitelisting on `k` here, and just allow `security.capability` and `user` prefixes for the moment.